.github/workflows/osl_sync_upstream.yml

```diff
@@ -0,0 +1,29 @@
+name: "CI :: OSL :: Sync Upstream"
+
+on:
+  workflow_dispatch: # Allows you to run the workflow manually from the Actions tab
+  schedule:
+    - cron: "0 2 * * 1" # Runs every Monday at 2 AM UTC (adjust as needed)
+
+env:
+  UPSTREAM_URL: https://github.com/apache/incubator-kie-tools
```

Comment:
`https://github.com/apache/incubator-kie-tools` will hopefully change soon, I would suggest using Workflow variables which we can reuse in other GHA.